dpnp/backend/extensions/statistics/sliding_window1d.cpp

```diff
@@ -30,20 +30,21 @@
 #include "utils/type_dispatch.hpp"
 #include <pybind11/pybind11.h>
 
+#include "ext/validation_utils.hpp"
 #include "sliding_window1d.hpp"
-#include "validation_utils.hpp"
 
 namespace dpctl_td_ns = dpctl::tensor::type_dispatch;
 using dpctl::tensor::usm_ndarray;
 using dpctl_td_ns::typenum_t;
 
+using ext::validation::array_names;
+using ext::validation::array_ptr;
+using ext::validation::check_num_dims;
+using ext::validation::common_checks;
+using ext::validation::name_of;
+
 namespace statistics
```

Comment:
can be combined then?
```suggestion
namespace statistics::sliding_window1d
```

Reply:
done